tests/createStringMeasurer.test.js

```diff
@@ -1,10 +1,131 @@
 const { createStringMeasurer } = require("../");
 
-test("measures an empty string", () => {
-  const size = createStringMeasurer([], 400);
+const testBody = ({ content, within, expectedWidth, expectedHeight, ...textStyles }) => {
+  const { width, height } = createStringMeasurer([{ content, textStyles }], within);
 
-  expect(size).toEqual({
-    height: 15,
-    width: 0
+  // TODO(lordofthelake): Sketch uses round numbers for widths.
+  // Considering that these are pixels, would it make sense to return rounded numbers too?
+  const roundedUpWidth = Math.ceil(width);
```

Comment:
it is `ceil` or `round` then?

Reply:
Ceil. I figured it out experimentally and it makes sense from a logical standpoint. If we're occupying "half a pixel", at a minimum we can paint one whole pixel (anti-aliasing notwithstanding).